sqlstore/src/main/resources/db/migration/V8886__change_seq_params.sql

```diff
@@ -4,6 +4,10 @@ UPDATE SequencingParameters SET chemistry = 'V2' WHERE platformId IN (SELECT pla
 UPDATE SequencingParameters SET chemistry = 'V3' WHERE xpath LIKE '%v3%';
 UPDATE SequencingParameters SET chemistry = 'V4' WHERE xpath LIKE '%v4%';
 UPDATE SequencingParameters SET chemistry = 'RAPID_RUN' WHERE xpath LIKE '%Rapid%';
-UPDATE SequencingParameters SET chemistry = 'V4' WHERE platformId IN (SELECT platformId FROM Platform WHERE name = 'ILLUMINA' AND instrumentModel LIKE '%NextSeq%');
+INSERT INTO SequencingParameters (name, platformId, readLength, paired, createdBy, creationDate, updatedBy, lastUpdated, chemistry)
+  SELECT CONCAT('High', name), platformId, readLength, paired, createdBy, CURRENT_TIMESTAMP, updatedBy, CURRENT_TIMESTAMP, 'NS_HIGH'
+  FROM SequencingParameters WHERE platformId IN (SELECT platformId FROM Platform WHERE name = 'ILLUMINA' AND instrumentModel LIKE '%NextSeq%') AND chemistry IS NULL AND name NOT LIKE 'Custom%';
+
+UPDATE SequencingParameters SET name = CONCAT('Mid ', name), chemistry = 'NS_MID' WHERE platformId IN (SELECT platformId FROM Platform WHERE name = 'ILLUMINA' AND instrumentModel LIKE '%NextSeq%') AND chemistry IS NULL AND name NOT LIKE 'Custom%';
```

Comment:
Running this migration doesn't change the values in my SequencingParameters table; I think the default for chemistry is `UNKNOWN` and not `NULL`.